apps/i18n/common/en_us.json

```diff
@@ -772,6 +772,8 @@
   "enableMakerDialogDescription": "Maker Toolkit is a feature used in the Computer Science Discoveries curriculum. See the setup page for more details:",
   "enableMakerDialogSetupPageLinkText": "Maker Toolkit Setup",
   "enablePairProgramming": "Allow students to Pair Program?",
+  "enablePeerFeedback": "Enable Peer Feedback?",
```

Comment:
yeah I'd also defer to @moneppo but one thought here is that "Peer Feedback" seems to describe the ability for students to comment on each others' projects (which is what is being enabled/disabled here) vs "Code Review" describes the whole review tab, including teacher comments?

Reply:
Also realizing I'm [saying conflicting things](https://github.com/code-dot-org/code-dot-org/pull/42069#discussion_r689942024) now :) I misunderstood on the other PR and thought the flag was enabling/disabling the Code Review feature entirely (rather than just commenting on peers' projects) but seeing how it's used here, "Peer Feedback" seemed to make more sense. So that being said, since the migration already uses "Code Review" i think sticking to "code review" for all the flags/parameters while using "Peer Feedback" in the user-facing component is okay?

Reply:
Let's use the spec'd text for UI components for now, but keep the code consistent. I've put a task on the product sprint to take a final pass of all the copy for the Code Review feature to ensure we're using consistent language.

Reply:
Task: https://codedotorg.atlassian.net/browse/CSA-710

Reply:
Sorry for not bringing up the question more broadly 😞 I tried to keep the code consistent and just this copy uses "peer feedback".

FWIW, I could definitely change all the variables to be `peer_feedback_enabled` (or the like) but I'm not sure how easy the column would be to change.

Reply:
Personally, having i18n keys similar to their values makes things simpler for me when I'm reading code. The key references a string in i18n whereas variables and columns reference features. Neither of those are strong opinions.